src/media/stream.ts

```diff
@@ -5,8 +5,15 @@ export enum StreamEventNames {
   Ended = 'stream-ended',
 }
 
+export enum MuteStateChangeReason {
```

Comment:
A bit of a nitpick, but I wonder if we should re-phrase this as "MuteStateChangeSource" and the values be more "plain nouns" (`User`, `Browser`).